librz/include/rz_asm.h

```diff
@@ -132,7 +132,7 @@ typedef struct rz_asm_plugin_t {
 	void *user; // user data pointer
 	int bits;
 	int endian;
-	bool (*init)(void *user);
+	bool (*init)(void **user);
```

Comment:
Given the PR is huge... why this change?

Reply:
because i need to save a context while the disassembler (or the analysis) runs. that user pointer was RzCore and makes no sense to have it there.